service/src/test/java/uk/gov/hmcts/reform/fpl/service/HearingBookingServiceTest.java

```diff
@@ -263,6 +263,17 @@ void shouldPersistJudgeWhenHearingIsUsingAlternateJudge() {
         assertThat(judgeAndLegalAdvisor.getLegalAdvisorName()).isEqualTo("Joe Bloggs");
     }
 
+    @Test
+    void shouldReturnTrueWhenFutureHearingExists() {
```

Comment:
Could we also have a shouldReturnFalseIf...version

Reply:
Ah yes I just pushed this I thought i'd written it already haha!